### ISSUE TITLE
Autodoc errors for documentation of interfaces/dashboard and resources/radiation_daysim/plot_points

### ISSUE THREAD

**Issue by Jack-Hawthorne** [architecture-building-systems/CityEnergyAnalyst#1729]
This error is thrown up by autodoc when running the make html command for documentation of the following:

    - interfaces/dashboard

```
WARNING: autodoc: failed to import module u'dashboard' from module u'cea.interfaces.dashboard'; the following exception was raised:
Traceback (most recent call last):
  File "C:\Users\Jack\Miniconda2\envs\cea\lib\site-packages\sphinx\ext\autodoc\importer.py", line 154, in import_module
    __import__(modname)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\interfaces\dashboard\dashboard.py", line 18, in <module>
    zone_building_names = locator.get_zone_building_names()
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\inputlocator.py", line 588, in get_zone_building_names
    zone_building_names = sorted(gdf.from_file(self.get_zone_geometry())['Name'].values)
TypeError: 'Mock' object has no attribute '__getitem__'
```
    - resources/radiation_daysim/plot_points

```
WARNING: autodoc: failed to import module u'plot_points' from module u'cea.resources.radiation_daysim'; the following exception was raised:
Traceback (most recent call last):
  File "C:\Users\Jack\Miniconda2\envs\cea\lib\site-packages\sphinx\ext\autodoc\importer.py", line 154, in import_module
    __import__(modname)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\resources\radiation_daysim\plot_points.py", line 14, in <module>
    print file[['Qhsf_MWhyr', 'Ef_MWhyr','Qcsf_MWhyr']].sum(axis = 0 )
TypeError: 'Mock' object has no attribute '__getitem__'
```

Looks like the Mock class in the docs/conf needs a __getitem__ attribute assigned to it.

@daren-thomas any ideas or suggestions?

**Comment by Jack-Hawthorne** [comment]
to replicate go to branch 1729 and run `make html` in the docs repos

**Comment by daren-thomas** [comment]
assuming https://readthedocs.org/projects/city-energy-analyst/builds/8436340/ runs through, then the course of action is to

- remove `geopandas` form `docs/config.py` mock-list
- add `geopandas` to the `docs/environment.yml` file.

see branch `origin/readthedocs` for an example of how to do that.

**Comment by daren-thomas** [comment]
it seems that the above suggestion really does work.

**Comment by Jack-Hawthorne** [comment]
@daren-thomas following your recommendation, i have removed `geopandas` from the mock-list and added to `environment.yml` however i still get the following error:

````
WARNING: autodoc: failed to import module u'benchmark' from module u'cea.analysis'; the following exception was raised:
Traceback (most recent call last):
  File "c:\users\jack\miniconda2\envs\cea\lib\site-packages\sphinx\ext\autodoc\importer.py", line 154, in import_module
    __import__(modname)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\analysis\benchmark.py", line 11, in <module>
    from geopandas import GeoDataFrame as gpdf
  File "c:\users\jack\miniconda2\envs\cea\lib\site-packages\geopandas\__init__.py", line 1, in <module>
    from geopandas.geoseries import GeoSeries
  File "c:\users\jack\miniconda2\envs\cea\lib\site-packages\geopandas\geoseries.py", line 22, in <module>
    class GeoSeries(GeoPandasBase, Series):
TypeError: Error when calling the metaclass bases
    metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases
``` 

This happens for most of the modules, any idea how to resolve the conflict?

**Comment by Jack-Hawthorne** [comment]
this happens for most modules unfortunately. to replicate go to branch `1729-autodoc-error-radiation-daysim`

**Comment by daren-thomas** [comment]
@Jack-Hawthorne I'm taking a look at this right now

**Comment by daren-thomas** [comment]
hey @Jack-Hawthorne can you pull this branch from origin? i think it's working now. Could you create a PR?

(btw: i just deleted `plot_points.py` - I don't think we need to keep it)

**Comment by daren-thomas** [comment]
wait... I'll create the PR.
